src/main/java/org/project_kessel/relations/client/CheckClient.java

```diff
@@ -1,8 +1,8 @@
 package org.project_kessel.relations.client;
 
-import org.project_kessel.api.relations.v0.KesselCheckServiceGrpc;
-import org.project_kessel.api.relations.v0.CheckRequest;
-import org.project_kessel.api.relations.v0.CheckResponse;
+import build.buf.gen.kessel.relations.v1beta1.CheckRequest;
```

Comment:
I'm not sure if there is any guidance on this package naming, or whether it is generated for you from the buf repo name or something, but in an ideal world we would probably want to preserve the full project kessel package name in the package, i.e.
`build.buf.gen.org.project_kessel.api.relations.v1beta1.*`

Reply:
Like above the package name is generated from the buf repo using the package name in the proto files ([kessel.relations.v1](https://github.com/project-kessel/relations-api/blob/68cda88c91435e7696c565a680311f508ccf2aa4/api/health/v1/health.proto#L3)). It may be possible to update it to something like `build.buf.gen.api.kessel.relations.v1beta1.* ` by changing the package in each proto to `package api.kessel.relations.v1beta1;`

I'm trying to see if there's another option with having to affect the proto files themselves

Reply:
We can do `org.project_kessel.api.kessel.relations.v1beta1` but it would require using `buf generate` and keeping the generated java classes in source control, which we don't want...

There's a couple different modes for [code generation](https://buf.build/docs/generate/overview) we're currently using generated SDKs which package every up for us to consume, but can't make it `project_kessel.api.relations.v1beta1` unless we make the directory structure the same which would be effectively making the protos stored in relations-api repo as such: `project_kessel/api/relations/v1beta1`. Right now its `api/kessel/relations/v1beta1`.